### ISSUE TITLE
RP issue

### ISSUE THREAD

**Issue by downboy** [elBukkit/MagicPlugin#259]
Some of players are having issues when they are trying to join a server where we have magic is installed. I will explain what is happening because there are no errors in console.

A player joins the server, they are in for like 1 or 2 seconds on this moment they should load the resource pack but there game freeze and are disconnected. I have removed magic plugin and then they can join again. So I think it had to do with magic plugin.
1 of the players in from island. I don't know if this is the issue. 

If you need more info you can always ask.

Greetings

**Comment by NathanWolf** [comment]
The RP can cause some clients to hang. There is unfortunately not much you can do about it other than turn off the RP or tell them to turn it off. They may have more success downloading it and installing it manually.

You could try increasing the rp load delay in config.yml, it's set to 2 seconds by default, sometimes it helps to wait a bit until their client has fully connected, loaded chunks, etc- but it's not a great user experience for the RP to load some long time after joining.

tl;dr it's a client problem that you can't really fix, it's not the plugin but the RP
